Examples/Algorithms/Fatras/include/ActsExamples/Fatras/FatrasAlgorithm.hpp

```diff
@@ -69,18 +69,18 @@ class FatrasAlgorithm final : public BareAlgorithm {
     // prepare output containers
     SimParticleContainer::sequence_type particlesInitialUnordered;
     SimParticleContainer::sequence_type particlesFinalUnordered;
-    SimHitContainer::sequence_type hitsUnordered;
+    SimHitContainer::sequence_type simHitsUnordered;
     // reserve appropriate resources
-    constexpr auto meanHitsPerParticle = 16u;
+    constexpr auto kMeanHitsPerParticle = 16u;
```

Comment:
What if we run this with sPhenix with ~100 hits per particle?

Reply:
Then the container reallocates. This is just a performance optimization and not critical to the function. The goal is to allocate sufficient memory once so in typical cases, the container does not need to reallocate again.